dinersclub/go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/nandanrao/chance v0.0.2
 	github.com/stretchr/testify v1.6.1
 	github.com/vlab-research/botparty v0.0.0-20200917171250-abbbd36eb095
-	github.com/vlab-research/go-reloadly v0.0.13
+	github.com/vlab-research/go-reloadly v0.0.16
```

Comment:
It may look strange to jump to `v0.0.16`, but while experimenting I created `0.14` and `0.15`, and it seems that once you've created a new GH release, the _contents_ of that released gets cached!

Reply:
Sure, that's normal to me, versions get skipped because some intermediates ones just weren't that good :D